payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncher.kt

```diff
@@ -124,10 +131,20 @@ interface PaymentLauncher {
 
             return remember(publishableKey, stripeAccountId) {
                 PaymentLauncherFactory(
-                    context,
-                    activityResultLauncher
+                    context = context,
+                    hostActivityLauncher = activityResultLauncher,
+                    statusBarColor = { context.findActivity()?.window?.statusBarColor },
                 ).create(publishableKey, stripeAccountId)
             }
         }
     }
 }
+
+private fun Context.findActivity(): Activity? {
```

Comment:
I feel like we have a few variants of this already, can we consolidate them?

Reply:
There’s only one in `identity` that we could consolidate this with. I think it’s fine as-is.